src/CommandLine/CommandLineParser.cpp

```diff
@@ -690,20 +701,24 @@ bool CommandLineParser::parseCommandLine(int argc, const char** argv) {
         m_debugCache = true;
       } else if (all_arguments[i] == "vpi_ids") {
         m_showVpiIDs = true;
-      } else {
-        int debugLevel = atoi(all_arguments[i].c_str());
+      } else if (all_arguments[i] == "coverelab") {
+        // Ignored!
+      } else if (is_number(all_arguments[i])) {
+        int debugLevel = std::stoi(all_arguments[i]);
```

Comment:
If we use `std::from_chars()` we can be compatible with a future in which all_arguments contains string-views.

Reply:
That future will likely need more changes than just swapping `std::stoi` to `std::from_chars()`. I switched to `std::stoi` because cl compiler then to warn about use of what it considers _unsafe_ functions.